modules/cluster-logging-deploy-cli.adoc

```diff
@@ -181,7 +181,11 @@ There should be an Elasticsearch Operator in each Namespace. The version number
 
 . Install the Cluster Logging Operator by creating the following objects:
 
+<<<<<<< HEAD
```

Comment:
@bobfuru Looks like there was a merge conflict that accidentally got committed here

Reply:
Thanks! - fixed in https://github.com/openshift/openshift-docs/pull/28819.